apiserver/facades/agent/uniter/networkinfo_test.go

```diff
@@ -297,8 +297,15 @@ func (s *networkInfoSuite) TestNetworksForRelationWithSpaces(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 
 	c.Assert(boundSpace, gc.Equals, spaceID3)
-	c.Assert(ingress, gc.DeepEquals,
-		network.SpaceAddresses{network.NewSpaceAddress("10.2.3.4", network.WithScope(network.ScopeCloudLocal))})
+
+	exp := network.SpaceAddresses{network.NewSpaceAddress(
```

Comment:
Ingress addresses are now fully formed, because they are not synthetically re-created from prematurely materialised `NetworkInfoResult`.